src/org/labkey/test/tests/list/ListDateAndTimeTest.java

```diff
@@ -0,0 +1,1368 @@
+package org.labkey.test.tests.list;
+
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.jetbrains.annotations.Nullable;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.SimplePostCommand;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.SortDirection;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.categories.Data;
+import org.labkey.test.categories.Hosting;
+import org.labkey.test.components.CustomizeView;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.domain.DomainFormPanel;
+import org.labkey.test.pages.ImportDataPage;
+import org.labkey.test.pages.core.admin.LookAndFeelSettingsPage;
+import org.labkey.test.pages.list.EditListDefinitionPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.util.DataRegionExportHelper;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.ExcelHelper;
+import org.labkey.test.util.ListHelper;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+@Category({Daily.class, Data.class, Hosting.class})
+public class ListDateAndTimeTest extends BaseWebDriverTest
+{
+
+    private static final String PROJECT_NAME = "List Date And Time Test";
+    private SimpleDateFormat _defaultDateFormat = new SimpleDateFormat("yyyy-MM-dd");
+    private SimpleDateFormat _defaultTimeFormat = new SimpleDateFormat("HH:mm:ss");
+    private SimpleDateFormat _defaultDateTimeFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm");
```

Comment:
It is misleading to give these initial values since they get defined in `doSetup`. They also need to be static for tests to see the value set in `doSetup` (each `@Test` is run with a different instance of the test class).
```suggestion
    // Fetched from look and feel page in 'doSetup'
    private static SimpleDateFormat _defaultDateFormat = null;
    private static SimpleDateFormat _defaultTimeFormat = null;
    private static SimpleDateFormat _defaultDateTimeFormat = null;
```

Reply:
Changed.